layers/+source-control/version-control/funcs.el

```diff
@@ -59,7 +59,7 @@
   (let ((current-prefix-arg t))
     (call-interactively
      (cl-case version-control-diff-tool
-       (diff-hl     'diff-hl-margin-minor-mode)
+       (diff-hl     'diff-hl-margin-mode)
```

Comment:
This would also check if we are in the terminal or not. 


Rich






On Mon, Jan 18, 2016 at 6:59 AM -0800, "Fabien Dubosson" <notifications@github.com> wrote:












In layers/+source-control/version-control/funcs.el:
> @@ -59,7 +59,7 @@
>    (let ((current-prefix-arg t))
>      (call-interactively
>       (cl-case version-control-diff-tool
> -       (diff-hl     'diff-hl-margin-minor-mode)
> +       (diff-hl     'diff-hl-margin-mode)




It's not autoloaded, which seems to be on purpose. Any idea why this function is used and not the plain mode?




It has been introduced here.



@ralesi Did you use diff-hl-margin-mode in purpose, or would diff-hl-mode be fine? (ref. #4034)



—
Reply to this email directly or view it on GitHub.